plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Blobstores.js

```diff
@@ -146,12 +151,7 @@ Ext.define('NX.coreui.controller.Blobstores', {
     var me = this,
         win = form.up('nx-coreui-blobstore-add');
 
-    if (win) {
-      me.loadStoreAndSelect(action.result.data.id, true);
-    }
-    else {
-      me.loadStore(Ext.emptyFn);
-    }
+    me.getStore('NX.coreui.store.Blobstore').load();
```

Comment:
Not a biggie, but we should strive for consistency here - some of these changes reflect where we use a ref for the store and access through the generated setter. For the record I prefer this style as the IDE can navigate directly to the Store impl, while the getter does not.